specification/managednetworkfabric/resource-manager/Microsoft.ManagedNetworkFabric/preview/2022-01-15-privatepreview/AccessControlLists.json

```diff
@@ -0,0 +1,582 @@
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Lists all AccessControlLists under the subscription.",
+            "schema": {
+              "$ref": "#/definitions/AccessControlListsListResult"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/ErrorResponse"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        },
+        "x-ms-examples": {
+          "AccessControlLists_ListBySubscription_MinimumSet_Gen": {
+            "$ref": "./examples/AccessControlLists_ListBySubscription_MinimumSet_Gen.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AccessControlListProperties": {
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./common.json#/definitions/AnnotationResource",
+          "description": "Common properties for annotated type."
+        }
+      ],
+      "properties": {
+        "addressFamily": {
+          "type": "string",
+          "description": "IP address family. Example: ipv4 | ipv6.",
+          "enum": [
+            "ipv4",
+            "ipv6"
+          ],
+          "x-ms-enum": {
+            "modelAsString": true,
+            "name": "addressFamily"
+          }
+        },
+        "conditions": {
+          "type": "array",
+          "description": "Access Control List conditions.",
+          "x-ms-identifiers": [
+            "sequenceNumber"
+          ],
+          "items": {
+            "description": "Access Control List condition model.",
+            "type": "object",
+            "allOf": [
+              {
+                "$ref": "./common.json#/definitions/AnnotationResource",
+                "description": "Common properties for annotated type."
+              }
+            ],
+            "properties": {
+              "sequenceNumber": {
+                "type": "integer",
+                "format": "int32",
+                "description": "sequenceNumber of the Access Control List."
+              },
+              "action": {
+                "type": "string",
+                "description": "action. Example: allow | deny.",
+                "enum": [
+                  "allow",
+                  "deny"
+                ],
+                "x-ms-enum": {
+                  "modelAsString": true,
+                  "name": "conditionActionType"
+                }
+              },
+              "destinationAddress": {
+                "type": "string",
+                "description": "destinationAddress. Example: any | 1.1.1.0/24 | 1.1.10.10"
```

Comment:
>destinationAddress [](http://example.com/codeflow?start=32&length=18)

It is good to generate Description with normal sentenses, not using variable names e.g. instead of destinationAddress, use Destination Address.

It will make your APIs read better in SDKs, CLIs etc. to your customers.

Same is applicable for all the descriptions in your swagger specs.